docroot/themes/humsci/su_humsci_theme/scss/base/_global.scss

```diff
@@ -98,15 +149,17 @@ div[data-contextual-id^="entityqueue"] {
 }
 
 // Emphasized text style
-.emphasized-text {
+.emphasized-text,
+.hs-emphasized-text {
```

Comment:
I've always wondered why this class wasn't prefixed with `decanter` like the other WYSIWYG classes. :smile: I think this makes more sense.

Reply:
yay!